telemetry/definitions/commonDefinitions.json

```diff
@@ -519,6 +519,18 @@
             "allowedValues": ["java", "python", "javascript", "plaintext", "jsx", "typescript", "tsx", "csharp",
                 "c", "cpp", "go", "kotlin", "php", "ruby", "rust", "scala", "shell", "sql"]
         },
+        {
+            "name": "codewhispererGettingStartedTaskType",
```

Comment:
Do not suffix "Type" at the end of types. 

More generally: please use existing code as a hint about conventions.

Reply:
Yeah, I did refer this as shown below but I can change accordingly. Thank you for the suggestion.

![image](https://github.com/aws/aws-toolkit-common/assets/141967714/04d8fd95-6644-4b96-98c3-e9d08babc190)